legal-copy/legal-templates.js

```diff
@@ -34,7 +32,7 @@ module.exports.templateEnd = `
           <a href="https://www.mozilla.org" class="mozilla-logo"></a>
           <a data-l10n-id="footerLinkLegal" href="https://www.mozilla.org/about/legal/" class="boilerplate">Legal</a>
 <a data-l10n-id="footerLinkPrivacy" href="/privacy" class="boilerplate">Privacy</a>
-<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms of Use</a>
+<a data-l10n-id="footerLinkTerms" href="/terms" class="boilerplate">Terms</a>
```

Comment:
Done.

@lmorchard @meandavejustice Any last objections to any of this nonsense before I merge?